test/addons-abi/3_callbacks/binding.cc

```diff
@@ -41,8 +41,7 @@ void Init(napi_env env, napi_value exports, napi_value module) {
     { "RunCallback", RunCallback },
     { "RunCallbackWithRecv", RunCallbackWithRecv }
   };
-  status = napi_define_properties(
-    env, exports, sizeof(desc) / sizeof(napi_property_descriptor), desc);
+  status = napi_define_properties(env, exports, 2, desc);
```

Comment:
if its always an array passed in, could we have a convenience macro for this that passes in the count- also, super-nitpicky, but generally, APIs that take in an array and count, take the array parameter first